Xamarin.Essentials/Clipboard/Clipboard.netstandard.cs

```diff
@@ -1,14 +1,17 @@
-﻿using System.Threading.Tasks;
+﻿using System.Runtime.InteropServices;
+using System.Threading.Tasks;
 
 namespace Xamarin.Essentials
 {
     public static partial class Clipboard
     {
-#if GTK
-        private static readonly Gdk.Atom clipboardAtom = Gdk.Atom.Intern("CLIPBOARD", false);
+        static readonly Gdk.Atom clipboardAtom = Gdk.Atom.Intern("CLIPBOARD", false);
 
         static Task PlatformSetTextAsync(string text)
         {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                throw ExceptionUtils.NotSupportedOrImplementedException;
```

Comment:
@webwarrior-ws Taras sorry but this is unacceptable because it will throw (crash) in UWP/Windows/WinUI , the idea was to only call GTK in the case of Linux, but run the existing code in case it's not Linux